src/main/java/muramasa/antimatter/datagen/providers/AntimatterBlockStateProvider.java

```diff
@@ -74,6 +84,10 @@ public AntimatterBlockModelBuilder getBuilder(Block block) {
         return (AntimatterBlockModelBuilder) models().getBuilder(block.getRegistryName().getPath());
     }
 
+    public AntimatterBlockModelBuilder getActiveBuilder(Block block) {
+        return (AntimatterBlockModelBuilder) models().getBuilder(block.getRegistryName().getPath() + "_active");
+    }
+
```

Comment:
This should not be needed, if you need to specify an active version of a model, use `existing`

Reply:
what do you mean by existing?